ldi-orchestrator/ldio-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioMetricValues.java

```diff
@@ -0,0 +1,8 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+public class LdioMetricValues {
+	public static final String PIPELINE = "pipeline";
+	public static final String LDIO_DATA_IN = "ldio_data_in";
+	public static final String LDIO_DATA_OUT = "ldio_data_out";
+	public static final String LDIO_COMPONENT_NAME = "ldio_type";
```

Comment:
Why don't we provide this in the sender metrics? I know it would always be the same sender component, but if we ever remove the sender or if we decide to keep metrics for the individual output components as well. Then we won't have to change the formatting of the data_out metrics. It would make it consistent with the input as well.